### PR TITLE
search: fix negative Content size

### PR DESCRIPTION
**Pull request by stefanhengl**
The GraphQL field `contentByteSize` is defined as `Int` which is not enough
to represent content size of very big repositories. For some customers
and for our megarepo we see negative content size on the index status
page in site admin view.

I had to make changes in frontend as well, because frontend uses a
function `prettyBytes` to format the output but unfortunately it only
takes `number` as input. Hence I wrote a little custom function.

Once this PR is in a good shape we should also convert other fields to
BigInt.

### CODE REVIEW

**Comment by stefanhengl** [comment]
@umpox I tagged you for the frontend parts. Let me know if there is better to way to work around the limitations of of pretty-bytes.

File: client/web/src/repo/settings/RepoSettingsIndexPage.tsx [hunk 1]
```diff
@@ -119,6 +119,19 @@ interface State {
     error?: Error
 }
 
+function prettyBytesBigint(bytes: bigint):string {
```
Comment (tsenart) [inline client/web/src/repo/settings/RepoSettingsIndexPage.tsx:122]:
```suggestion
function prettyBytesBigint(bytes: bigint): string {
```

**Review by tsenart — APPROVED** [review]

**Comment by tsenart** [comment]
@stefanhengl: Prettier is still sad. Also, if CI passes, I wouldn't block merge on frontend review since this is a pretty isolated change. There aren't automated deployments going out though so we'd have to deploy manually if needed.

**Comment by tsenart** [comment]
LGTM

**Pull request merged**